docs/common/sources.rst

```diff
@@ -17,3 +17,4 @@ The expected parameter for the VM images is ``vmi_build``:
 
 .. _KojiSource: https://release-engineering.github.io/pushsource/sources/koji.html#accessing-virtual-machine-images
 .. _ErrataSource: https://release-engineering.github.io/pushsource/sources/errata.html
```

Comment:
I forgot last time, can you please add an example of using the staged source?

We have the example for koji only.

The rest LGTM

Reply:
TYVM